web/src/components/googleMap.js

```diff
@@ -1,7 +1,11 @@
 import React, { Component } from 'react';
 import { withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
+import axios from 'axios';
 import LocationInput from './LocationInput';
 
+const Key = '&key=AIzaSyDFPbV-j4fvg70Ziq2bbTK_k5JSsfyidac';
```

Comment:
Is this an API key? Suggestion: You might want to create a variable else where and save it in your gitignore so your key isnt available publicly.